<ui version="4.0" >
 <class>StoreCoverDialog</class>
 <widget class="QDialog" name="StoreCoverDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>420</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Store cover</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="checkBox" >
        <property name="text" >
         <string>Scale before store</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Horizontal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="value" >
           <number>300</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Vertical</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_2" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="value" >
           <number>300</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox" >
          <property name="minimumSize" >
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Cover mask</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit" />
        </item>
        <item>
         <widget class="QToolButton" name="toolButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../audex.qrc" >:/buttons/images/wizard.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../audex.qrc" />
 </resources>
 <connections/>
</ui>
